packages/vertexai/src/models/vertexai-model.ts

```diff
@@ -78,6 +78,8 @@ export abstract class VertexAIModel {
         apiKey: vertexAI.app.options.apiKey,
         project: vertexAI.app.options.projectId,
         appId: vertexAI.app.options.appId,
+        automaticDataCollectionEnabled:
```

Comment:
No action needed, just wondering out loud if at some number of properties, we just pass vertexAI.app or at least vertexAI.app.options through instead of passing them through one by one.

Reply:
agreed- i could try this out